hapi-fees/src/main/java/com/hedera/services/usage/TxnUsage.java

```diff
@@ -26,13 +26,13 @@
 
 public abstract class TxnUsage {
     protected static final int AMOUNT_REPR_BYTES = 8;
-    protected static EstimatorFactory estimatorFactory = TxnUsageEstimator::new;
+    private static EstimatorFactory estimatorFactory = TxnUsageEstimator::new;
```

Comment:
A mutable static field that is only updated in tests is a major red flag to me, one that we shouldn't just paper over with getters and setters if there are better alternatives.

I think we should investigate if we should move these to injected objects in the places they are used (such as `TokenAssociateResourceUsage`, which is already Dagger injected) and then update the tests to manually inject their mocks.

Reply:
Removed the static field as you suggested and injecting instead.  This caused a chain of tests to have to be revisited.  I cleaned up some things that seemed inconsistent among the tests.

Reply:
Also netted 3 less lines after the refactor.